src/cumulus/cli.py

```diff
@@ -30,13 +30,17 @@
 
 from subprocess import call
 
+
 DOCKER_HUB = "strcum/"
 DOCKER_COMPOSE = "docker-compose"
 ENTRYPOINT = "./docker_entrypoint.sh"
 LOGFILE = "docker-compose-log.out"
 VERSION = '1.0.0'
 DATABASE = ["mysql", "postgres"]
 WEB_APP = ["django", "rails"]
+PORTS = {"django": "41000", "rails": "41001"}
+COMMANDS = {"django": "python manage.py runserver 0:"+PORTS["django"],
```

Comment:
Can we use .format for string formatting :D

Reply:
Yeps forgot.